src/rgw/rgw_op.h

```diff
@@ -181,6 +182,7 @@ class RGWOp : public DoutPrefixProvider {
   RGWQuotaInfo user_quota;
   int op_ret;
   int do_aws4_auth_completion();
+  jspan parent_op_trace;
```

Comment:
I added this to RGWOp because I thought that we can use it in other cases in which an Op leads to another Op.
otherwise, we can rename it and put it under `RGWInitMultipart`.